install.ps1

```diff
@@ -924,6 +924,14 @@ $backgroundImage = "${Env:VM_COMMON_DIR}\background.png"
 # Use background image for lock screen as well
 $lockScreenImage = "${Env:VM_COMMON_DIR}\lockscreen.png"
 Copy-Item $backgroundImage $lockScreenImage
+# Download FLARE VM (internet enabled) background image
+$backgroundInternetImage = "${Env:VM_COMMON_DIR}\background-internet.png"
```

Comment:
I wonder if it is possible to create a `VM` subfolder in `Images` and download the whole folder content to `${Env:VM_COMMON_DIR}` instead of downloading all of them individually like you are doing now, it allows us more flexibility if we need to change the images names or add more images. we would still need to duplicate the lockscreen image.